storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -368,29 +371,62 @@ static bool check_server_version(unsigned long version_number,
     server_flavor = FLAVOR_MYSQL;
   }
 
+  if (server_flavor == FLAVOR_MARIADB) {
+    xb::error()
+        << "This version of Percona XtraBackup does not support MariaDB";
+    return false;
+  }
+
+  DBUG_EXECUTE_IF("simulate_lower_version", version_number = 80499;);
+
   version_supported =
-      version_supported || (version_number > 80000 && version_number <= 80099);
+      version_supported || (version_number >= 80100 && version_number <= 80199);
```

Comment:
Please add  comment that PXB 8.1 supports Server versions 8.1 until 8.1.99

Reply:
Done